### PR TITLE
update middleware update notification

### LINKED ISSUE #7596: Add Appropriate Notification upon updating facility middleware configuration
**Describe the bug**

When updating the facility middleware address, ensure that the notification message accurately describes the action taken.

**To Reproduce**

1. Go to facility configuration page
2. update its middleware address
3. See error in the notification

**Expected behavior**

Upon successful update of the facility middleware, modify the notification to display the specific message: 'Facility Middleware updated successfully'.

**Screenshots**

image

### PR DESCRIPTION
**Pull request by rash-27**
## Proposed Changes

- Fixes #7596 
- Updated the middleware update notification

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers
- After modification
image

### CODE REVIEW

File: src/Components/Facility/FacilityConfigure.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ export const FacilityConfigure = (props: IProps) => {
     setIsLoading(false);
     if (res?.ok) {
       Notification.Success({
-        msg: "Facility updated successfully",
+        msg: "Facility middleware updated successfully",
```
Comment (bodhish) [inline src/Components/Facility/FacilityConfigure.tsx]: Lets move strings to `i18n`

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Components/Facility/FacilityConfigure.tsx (hunk 1 above)
Comment (rash-27) [reply src/Components/Facility/FacilityConfigure.tsx]: yeah sure

Comment (rash-27) [reply src/Components/Facility/FacilityConfigure.tsx]: @bodhish should we also keep translations in other languages  ?

**Review by rithviknishad — APPROVED** [review]

File: src/Components/Facility/FacilityConfigure.tsx (hunk 1 above)
Comment (rithviknishad) [reply src/Components/Facility/FacilityConfigure.tsx]: yes if possible, not mandatory.

Comment (rash-27) [reply src/Components/Facility/FacilityConfigure.tsx]: Okay

**Comment by nihal467** [comment]
LGTM

**Review by Ashesh3 — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Review by gigincg — APPROVED** [review]

**Pull request merged**